src/stories/InlineForm.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { Meta, StoryFn } from '@storybook/react';
+import styled from 'styled-components';
+import { IconQuestionCircleOutline } from '@taskany/icons';
+import { gray7 } from '@taskany/colors';
+
+import { InlineForm } from '../components/InlineForm';
+import { Button } from '../components/Button';
+import { TableCell } from '../components/Table';
+import { Form } from '../components/Form';
+import { Popup } from '../components/Popup';
+import { FormInput } from '../components/FormInput';
+
+export default {
+    title: 'InlineForm',
+    component: InlineForm,
+    argTypes: { onSubmit: { action: 'form submitted' } },
+} as Meta<typeof InlineForm>;
+
+const StyledWrapper = styled.div`
+    display: contents;
+`;
+
+const StyledForm = styled(Form)`
+    background-color: transparent;
+`;
+
+const StyledTableCell = styled(TableCell)`
+    flex-wrap: nowrap;
+    display: flex;
+    align-items: center;
+`;
+
+const StyledFormInput = styled(FormInput)`
+    font-size: 14px;
```

Comment:
does we really need this custom styles to make form example?

Reply:
Not needed. We can use default